extensions/amp-ad/0.1/amp-ad-api-handler.js

```diff
@@ -84,6 +88,9 @@ export class AmpAdApiHandler {
     this.iframe_ = iframe;
     this.is3p_ = is3p;
     this.iframe_.setAttribute('scrolling', 'no');
+    let iframeSrc = this.iframe_.getAttribute('src');
```

Comment:
Not sure if `getAttribute('src')` is ok. Can code inside iframe change this attribute?